src/ExNavigationBar.js

```diff
@@ -14,6 +14,8 @@ import {
 } from 'react-native';
 import PureComponent from 'PureComponent';
 
+import { Components } from 'exponent';
```

Comment:
For non-exponent apps, would it make sense to pass-in a BlurView as a `prop`? Then you could just use: https://github.com/react-native-community/react-native-blur if you wanted.